scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -791,7 +791,7 @@ <h6 class="card-subtitle mb-2 text-muted"><span class="badge badge-warning">Part
 {% macro dismissed_gene_list(variant) %}
   {% if variant.genes %}
     {% if variant.genes|length > 3%}
-      {{ variant.genes|length }}&nbsp;genes
+      {{ variant.hgnc_symbols|length }}&nbsp;genes
```

Comment:
Perphaps also add a small note in e.g. the controller or view docstring describing the difference so it's not confusing if one needs to change it later?

Reply:
Good idea, I'll fix!